cmd/kubelet/app/server.go

```diff
@@ -714,7 +714,6 @@ func RunKubelet(kubeFlags *options.KubeletFlags, kubeCfg *kubeletconfiginternal.
 		kubeFlags.RegisterNode,
 		kubeFlags.RegisterWithTaints,
 		kubeFlags.AllowedUnsafeSysctls,
-		kubeFlags.Containerized,
```

Comment:
This flag is useful. It is used to indicate that `kubelet` is running in containerized mode.

Reply:
It appears that this particular boolean in this particular code path is not used. This PR is not removing the code that configures the NsenterMounter, for example (https://github.com/kubernetes/kubernetes/blob/d1e711a6af5379f48be4ea585911977000be2af3/cmd/kubelet/app/server.go#L141-L145).

Reply:
I think @ncdc is right. This doesn't remove the flag, nor the field in the kubelet configuration. It only removes the redundant passing of this variable.

Reply:
Thanks @ncdc @yujuhong for pointing out. My bad.